2024-04-30 11:42:19.614581+02:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'a' (EMPLEADO: ""EMPRESA_ID"" BIGINT)"; SQL statement:
INSERT INTO "PUBLIC"."EMPLEADO"("ID","CORREO_ELECTRONICO","DEPARTAMENTO","EMPRESA_ID","ES_CONTROLADOR","ES_RESPONSABLE","NOMBRE_COMPLETO","NOMBRE_EMPRESA","NUMERO_TELEFONO","PASSWORD","PASSWORD_EMPRESA","PUESTO","SUSCRIPCION_EMPRESA")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?) [22018-224]
2024-04-30 11:42:19.614581+02:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "'a' (EMPLEADO: ""EMPRESA_ID"" BIGINT)"; SQL statement:
INSERT INTO "PUBLIC"."EMPLEADO"("ID","CORREO_ELECTRONICO","DEPARTAMENTO","EMPRESA_ID","ES_CONTROLADOR","ES_RESPONSABLE","NOMBRE_COMPLETO","NOMBRE_EMPRESA","NUMERO_TELEFONO","PASSWORD","PASSWORD_EMPRESA","PUESTO","SUSCRIPCION_EMPRESA")VALUES(?,?,?,?,?,?,?,?,?,?,?,?,?) [22018-224]
2024-04-30 11:47:41.815606+02:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM HORARIOS [*]AND AUSENCIAS"; SQL statement:
SELECT * FROM HORARIOS AND AUSENCIAS [42000-224]
2024-04-30 11:48:36.623509+02:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "U&'ma\\00f1ana' (HORARIOS: ""JORNADA"" BOOLEAN NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."HORARIOS"("ID","DIA","HORA_ENTRADA","HORA_SALIDA","JORNADA","MINUTOS_EXT","MINUTOS_PAU","MINUTOS_TOT","EMPLEADO_ID")VALUES(?,?,?,?,?,?,?,?,?) [22018-224]
2024-04-30 11:48:36.623509+02:00 jdbc[14]: exception
org.h2.jdbc.JdbcSQLDataException: Data conversion error converting "U&'ma\\00f1ana' (HORARIOS: ""JORNADA"" BOOLEAN NOT NULL)"; SQL statement:
INSERT INTO "PUBLIC"."HORARIOS"("ID","DIA","HORA_ENTRADA","HORA_SALIDA","JORNADA","MINUTOS_EXT","MINUTOS_PAU","MINUTOS_TOT","EMPLEADO_ID")VALUES(?,?,?,?,?,?,?,?,?) [22018-224]
2024-05-01 11:04:51.425169+02:00 jdbc[15]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "HORARIO" not found; SQL statement:
SELECT * FROM HORARIO [42102-224]
2024-05-01 11:04:52.497897+02:00 jdbc[15]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "HORARIO" not found; SQL statement:
SELECT * FROM HORARIO [42102-224]
2024-05-03 22:11:33.902746+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "USERS" no encontrada
Table "USERS" not found; SQL statement:
select username,password,enabled from users where username = ? [42102-224]
2024-05-03 22:11:47.657716+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "USERS" no encontrada
Table "USERS" not found; SQL statement:
select username,password,enabled from users where username = ? [42102-224]
2024-05-03 22:12:54.573392+02:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "USERS" no encontrada
Table "USERS" not found; SQL statement:
select username,password,enabled from users where username = ? [42102-224]
2024-05-05 12:25:27.654549+02:00 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Tabla "EMPLEADOS" no encontrada
Table "EMPLEADOS" not found; SQL statement:
SELECT * FROM EMPLEADOS [42102-224]
2024-05-05 13:13:58.006575+02:00 jdbc[25]: null
org.h2.message.DbException: La conexi贸n no fue cerrada por la aplicaci贸n y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-224]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:218)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:135)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:733)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: La conexi贸n no fue cerrada por la aplicaci贸n y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 16 more
